pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.black]
```

Comment:
Why didn't you add this to `.pre-commit-config.yaml`

Reply:
I've always preferred using `pyproject.toml` since it applies more broadly to the project's configuration, instead of it only being in pre-commit.  I guess if you wanted to run black outside of pre-commit it would still apply.

Why put it in .pre-commit-config.yml instead?

Reply:
Yeah, I don't have a strong opinion about it. I was curios your decision :)